pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -166,10 +200,43 @@ func (r *ReconcileWindowsMachineConfig) getCloudProvider(instance *wmcapi.Window
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileWindowsMachineConfig) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	log.Info("reconciling", "namespace", request.Namespace, "name", request.Name)
+	mayBeWMCOobject := false
```

Comment:
`wmcObject` please

Reply:
This variable name is confusing, maybe change it to `notMachineObject`, or `handleWMCObject`?